RATIONALE.md

```diff
@@ -13,21 +13,41 @@ Note: `microwasm` was never specified formally, and only exists in a historical
 https://github.com/bytecodealliance/wasmtime/blob/v0.29.0/crates/lightbeam/src/microwasm.rs
 
 
-## Size limitations
-### Number of functions
+## Implementation limitations
+
+[WebAssembly specification states that](https://www.w3.org/TR/wasm-core-1/#a2-implementation-limitations)
```

Comment:
WebAssembly 1.0 (MVP) specification allows runtimes to limit certain aspects of Wasm module or execution.